test/python/taskbasedsimulator/test_path_sim_qasm_simulator.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+
+from qiskit import QuantumCircuit, BasicAer
+from jkq.ddsim.pathqasmsimulator import PathQasmSimulator
+from qiskit import execute
+
+
+class JKQQasmSimulatorTest(unittest.TestCase):
+    """Runs backend checks and the Basic qasm_simulator tests from Qiskit Terra."""
+
+    def setUp(self):
+        self.backend = PathQasmSimulator()
+        self.circuit = QuantumCircuit.from_qasm_str('''OPENQASM 2.0;
+            include "qelib1.inc";
+            qreg q[3];
+            qreg r[3];
+            h q;
+            cx q, r;
+            creg c[3];
+            creg d[3];
+            barrier q;''')
```

Comment:
The standard version of this benchmark contains measurements (see tests for the other simulator backends). This should be consistent across the project.